cli.sh

```
@@ -2290,28 +2229,61 @@ install_local_skills() {
-		link_skill_to_targets "$skill_name" "$UNIVERSAL_SKILLS_DIR"
-	done
+		# Remove existing directory/symlink if it exists
+		if [ -e "$tool_dir" ] || [ -L "$tool_dir" ]; then
+			# Check if it's already correctly symlinked
+			if [ -L "$tool_dir" ] && [ "$(readlink "$tool_dir")" = "$universal_dir" ]; then
+				continue
+			fi
+			# Back up existing non-symlink directory to central backup location
+			# (outside tool config to avoid skill conflicts from duplicate scanning)
+			if [ -d "$tool_dir" ] && [ ! -L "$tool_dir" ]; then
+				local backup_dir="$HOME/.my-ai-tools-backups/skills/$(basename "$tool_dir").backup.$(date +%Y%m%d%H%M%S)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)